internal/hostport/fake_iptables.go

```diff
@@ -146,6 +146,7 @@ func (f *fakeIPTables) ensureRule(position utiliptables.RulePosition, tableName
 		return true, nil
 	}
 
+	// nolint:gocritic // using a switch statement is not much different
```

Comment:
It will actually look cleaner with switch:
```go
switch(position) {
  case utiliptables.Prepend:
    chain.rules = append([]string{rule}, chain.rules...)
  case utiliptables.Append:
    chain.rules = append(chain.rules, rule)
  default:
    return false, fmt.Errorf("unknown position argument %q", position)
}